libminifi/CMakeLists.txt

```diff
@@ -141,11 +141,14 @@ endif()
 SET (LIBMINIFI core-minifi PARENT_SCOPE)
 
 if (ENABLE_PYTHON)
-if (NOT APPLE)
 #### shared
 
 add_library(core-minifi-shared SHARED ${SOURCES})
-target_link_libraries(core-minifi-shared ${CMAKE_DL_LIBS} uuid-shared yaml-cpp)
+if (APPLE)
```

Comment:
I think uuid-shared is already linked elsewhere, so I guess it was just unnecessary?

Reply:
Indeed, it ended up with double definition errors. However works fine in Linux.

Reply:
Hmm. Maybe we should add -DENABLE_PYTHON to our travis builds

Reply:
I strongly agree, but in that case Python related tests (the ones that don't transport, just create instance, processor, flow files) could also be added to cover that.

Reply:
Indeed -- https://issues.apache.org/jira/browse/MINIFICPP-660 already exists. I've pre-empted that in favor of CoAP, but should get back to that soon. Python is kind of WIP. I do have some tests from that branch that cover what you are talking about. I'll try and re-surface them as soon as I get away from CoAP.